purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -93,6 +93,8 @@ internal class BillingWrapper(
     // how long before the data source tries to reconnect to Google play
     private var reconnectMilliseconds = RECONNECT_TIMER_START_MILLISECONDS
 
+    private var reconnectionAlreadyScheduled = false
```

Comment:
I guess the ideal way of doing this would be to have a queue of jobs, and check if it's empty... But this seemed like a simpler solution for now. Lmk what you think.